### DISCUSSION TITLE
баг с паладином в 4 главе

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2_classic#96] [General]
я при игре не встречал бага с пропажей паладина в минентале который торгует заклинаниями. но про это писали в стими по моему?

**Comment by dosinabox** [comment]
Да, в стиме писали. Изучить уже нельзя, у юзера не осталась сохраненки. Код на первый взгляд в порядке, не знаю в чем может быть проблема. Надежнее всего сделать нового нпц и частично переносить ему инвентарь, как сделано в обновлении г2нв.

**Reply by Kor-Angar** [reply]
если есть функция переноса есть то да. если функции переноса нет, то можно просто создать новому нпс пару зелий маны.

**Comment by Kor-Angar** [comment]
труп был. в трупе было полно товара, который я ему продал в 3 главе :-) я думаю будет лучше вставлять новую инстанцию как ты делала в неоф. обновление и давать пару свитков и эликсиров. опять же как это у тебя в неоф. обновление.

**Reply by Kor-Angar** [reply]
> я ему продал в 4 главе :-)

в 3 главе.

**Comment by Kor-Angar** [comment]
сенграт в 4 главе 
![20230307002452_1](https://user-images.githubusercontent.com/49514764/224621641-fed4b410-17fb-423d-bc8c-c253a8c1cad3.jpg)
